$OpenBSD: patch-props_ml,v 1.1 2005/05/30 19:27:45 sturm Exp $
Post-release developer patch.
o Fix in property setting.  In particular, set properties in the right order.
  See http://lists.seas.upenn.edu/pipermail/unison-hackers/2005-May/000091.html
--- props.ml.orig	Sat Mar 12 11:17:59 2005
+++ props.ml	Fri May 27 17:31:10 2005
@@ -421,7 +421,7 @@ let syncedPartsToString t = match t with
 
 let iCanWrite p =
   try
-    Unix.access p [Unix.R_OK];
+    Unix.access p [Unix.W_OK];
     true
   with
     Unix.Unix_error _ -> false
@@ -625,8 +625,8 @@ let set fspath path kind p =
   Uid.set fspath path kind p.uid;
   Gid.set fspath path kind p.gid;
   TypeCreator.set fspath path kind p.typeCreator;
-  Perm.set fspath path kind p.perm;
-  Time.set fspath path kind p.time
+  Time.set fspath path kind p.time;
+  Perm.set fspath path kind p.perm
 
 let init someHostIsRunningWindows =
   Perm.init someHostIsRunningWindows;
